luminoso_api/upload.py

```diff
@@ -103,9 +101,9 @@ def main():
         default=URL_BASE
     )
     parser.add_argument(
-        '-r', '--readers', metavar='LANG=READER',
-        help=("Custom reader to use, in a form such as "
-              "'ja=mecab.ja,en=freeling.en'")
+        '-l', '--language', metavar='LANG',
```

Comment:
What is this "metavar" for?  I think by default it will be "LANGUAGE" (the name of the argument), which is probably better.